context/package.json

```diff
@@ -57,7 +57,7 @@
     "rehype-raw": "^7.0.0",
     "sass": "^1.72.0",
     "searchkit": "^2.4.1-alpha.4",
-    "styled-components": "^5.3.10",
+    "styled-components": "^6.1.12",
```

Comment:
Do we still need `styled-components` as a main dependency?

Reply:
Yes; MUI requires either styled-components or emotion as a peer dependency. From what I found, I determined that updating SC is a lower-friction option than switching to emotion at this time (especially if we're able to go through with migrating to Pigment down the line once we upgrade to MUI 6 or 7).